### PR TITLE
Basic GraphQL Admin API

### PR DESCRIPTION
**Pull request by kiootic**

### CODE REVIEW

**Comment by louischan-oursky** [comment]
Looks good. How is the study on dataloader integration?

**Comment by kiootic** [comment]
I'll implement some connections (i.e. identities) before start investigating data loader.

By the way, we discussed on fixing the nullability of the edge & connection types. I just did a quick search and found that [nullable edges are intentional by spec](https://stackoverflow.com/a/57908071/13196616), it allows better error reporting.

**Comment by kiootic** [comment]
@louischan-oursky Add authorization check for admin API.

I'll fix #117 (without labels) before continuing to add identities & authenticators.

File: pkg/admin/loader/user.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ type UserLoader struct {
 }
 
 func (l *UserLoader) Get(id string) (interface{}, error) {
+	println("get user " + id)
```
Comment (louischan-oursky) [inline pkg/admin/loader/user.go:16]: Remove this line.

**Pull request merged**